doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's light multi-user chat commands set
+
+## Purpose
+This is a set of commands, which provide actions connected with multi-user chat (light) features.
```

Comment:
This is a set of commands, providing actions connected with multi-user chat (light) features?